src/apiary.apib

```diff
@@ -1544,6 +1544,61 @@ level `conversions` array.
                 + conversion (Conversion)
                     + conversion_type: 3 (number)
 
+# Group Alert
+
+Alerts are used across application to notify users (when subscribed) of events happening in platform.
+
+## Alerts [/alerts/alert]
+
+### Get All Available Alerts [GET]
```

Comment:
Am I right to say that you are looking this structure:
```
{
  "alerts": [
    {
      "code": "",
      "name": "",
      "description": "",
      "service": "",
      "id": 0
    }
  ]
}
```

... to be replaced with the following:

```
{
    "alerts": [{
        "alert": {
            "code": "",
            "name": "",
            "description": "",
            "service": "",
            "id": 1
        },
        "alert":{
            "code": "",
            "name": "",
            "description": "",
            "service": "",
            "id": 2
        }
    }]
}
```

This is how I understand your description, but it doesn't feel right, as the keys are not unique.